src/IMU.cpp

```diff
@@ -109,6 +114,58 @@ void IMUClass::end()
   _wire->end();
 }
 
+voidFuncPtr cb;
+IMUClass * imurq;
+void IMUClass::HighMotionAcceleration()
+{
+  // check if any motion interrupt was triggered;
+  if (imurq->readRegister(BNNO055_INT_STA_REG) == 0x40){
+    cb();
+  }
+
+  // clear the interupt status bit
+  imurq->writeRegister(BNNO055_SYS_TRIGGER_REG, 0x40);
+}
+
+void IMUClass::attachInterrupt(voidFuncPtr callback)
+{
+  cb = callback;
+  imurq = this;
+
+  // attach interrupt to pin 0
+  ::attachInterrupt(0, HighMotionAcceleration, RISING);
```

Comment:
Should the pin be passed in the constructor?

Reply:
yes it will be, i'll add a variable and set as usual on the constructor